internal/appsec/waf_test.go

```diff
@@ -167,7 +167,6 @@ func TestWAF(t *testing.T) {
 		event := finished[0].Tag("_dd.appsec.json")
 		require.NotNil(t, event)
 		require.Contains(t, event, "crs-942-270") // SQLi
-		require.Contains(t, event, "crs-941-100") // XSS
```

Comment:
can you please check if there's a new attack found instead of this one?
IIRC, we wanted more than one attack here for the sake of having a test a little-bit more advanced.

Reply:
There was no new attack, but I modified the payload to trigger another xss detection, see c06c7046